pkg/volume/cinder/cinder_util.go

```diff
@@ -132,6 +132,40 @@ func (util *CinderDiskUtil) DetachDisk(cd *cinderVolumeCleaner) error {
 	return nil
 }
 
+func (util *CinderDiskUtil) DeleteVolume(cd *cinderVolumeDeleter) error {
+	cloud := cd.plugin.host.GetCloudProvider()
+	if cloud == nil {
+		glog.Errorf("Cloud provider not initialized properly")
+		return errors.New("Cloud provider not initialized properly")
+	}
+
+	if err := cloud.(*openstack.OpenStack).DeleteVolume(cd.pdName); err != nil {
+		glog.V(2).Infof("Error deleting cinder volume %s: %v", cd.pdName, err)
+		return err
+	}
+	glog.V(2).Infof("Successfully deleted cinder volume %s", cd.pdName)
+	return nil
+}
+
+func (util *CinderDiskUtil) CreateVolume(c *cinderVolumeProvisioner) (string, int, error) {
+	cloud := c.plugin.host.GetCloudProvider()
+	if cloud == nil {
+		glog.Errorf("Cloud provider not initialized properly")
+		return "", 0, errors.New("Cloud provider not initialized properly")
+	}
+
+	// convert to GB with rounding
+	const giga = 1024 * 1024 * 1024
+	volSize := int((c.options.Capacity.Value() + giga - 1) / giga)
```

Comment:
> The code looks very similar to AWS, just with GB (AWS uses MB), but I think it will be quite different when we add QoS, as they are fundamentally different between various clouds.

How so? As far as I can tell this just converts bytes to GiB (not GB) with some rounding. Feels like there should be some existing common util function to do this type of conversion, and if one doesn't exist it'd be nice to add one (and have this AWS and GCE code use it).